README.md

```diff
@@ -369,4 +369,19 @@ jobs:
       dockerComposeComponents: 'postgres'
 ```````
 
+### test-python.yml 
+
+Supports optional `pythonVersionMatrix` input that allows the specificity of Python version (default 3.11)
```

Comment:
```suggestion
Supports optional `pythonVersionMatrix` input that allows the specificity of Python version. A default is defined in the workflow.
```

Suggest we just note there is a default defined, but leave it to the code to document what the value is, to save us having to update the value in multiple places when the LTS changes.